mp.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+DOCKER_IMAGES=`git grep Docker.Builder 2>/dev/null | sed "s/.*Docker.Builder//g" | cut -d, -f2 | cut -d\" -f2 | sort | uniq | grep -v Invalid`
+
+case "$1" in 
```

Comment:
Optional: adding `help` describing what each option exactly does

Reply:
I'm afraid script would be unnecessarily bloated, I think usage message covers this sufficiently

```shell
*)
   echo "Usage: $0 {dist|dist-zip|dist-run|save-images|load-images|ts-maven-repo|clean}"
esac
```

Target names are hopefully clear enough, each target has up to 4 command.

Reply:
I had to take a look at script to see difference between dist|dist-zip and why is here save-images|load-images.

Reply:
I think I will promote just 2 end-user facing commands